### DISCUSSION TITLE
Handling negative grid pricing

### DISCUSSION THREAD

**Discussion by ArrEssJay** [amberelectric/public-api#89] [General]
I'm seeing some negative prices in the forecast for tomorrow: $-1 /kWh. We don't have solar.

<img width="1457" alt="Screen Shot 2022-01-02 at 14 31 32" src="https://user-images.githubusercontent.com/3732357/147865320-9ca51f19-3a6e-41bd-9687-6ad375199da0.png">

The data on the amber website is showing the same thing.

As much as I'd be happy to be paid to find a large resistive load to dump as much power as our grid supply will handle into, that seems like an unlikely scenario.

Is there any info on where this forecast is coming from and what this means in practise?

**Comment by madpilot** [comment]
It's very possible, though unlikely.

Generally, future negative forecasts dissipate closer to the time, but there have definitely been times when our customers have been paid to use electricity.

We use the forecasts published by AEMO.

**Comment by edwapj** [comment]
Negative spot prices have become more frequent in regions with large amounts of solar power, as in South Australia and Victoria.

A useful resistive load is a storage hot water service which Amber can switch on for you during the lowest priced periods each day.     Or you can arrange the switching yourself behind the meter.

Last October, the General Usage Wholesale (Av. Price) for my household in SA was negative $2.27 for 715kWh  although the network and supply charges raised my total bill to $153,         This result was solely due to averaging down the import price using my hot water service as I have no solar panels.

Why I should have had to pay an environmental charge of $14 for using other peoples' surplus solar energy is not immediately clear to me.
